java-frontend/src/test/java/org/sonar/java/UCFGJavaVisitorTest.java

```diff
@@ -244,11 +428,13 @@ public void build_primitive_types_to_constants() {
         "    long var6 = 12L;\n" +
         "    float var7 = 1.0;\n" +
         "    double var8 = 1.0;\n" +
-        "    System.out.print(var5);\n" +
-        "    System.out.print(arg1);\n" +
+        "    foo(var5);\n" +
+        "    bar(arg1);\n" +
         "    String s = \"\" + var5 + var6 + var7 + var8;\n" +
         "    return s + arg1 + arg2 + arg3 + arg4;\n" +
         "  }\n" +
+        "  void foo(int i) {}\n" +
```

Comment:
or simply `void foo(Object o) {}\n` to be used by both.
But I would have kept one of the chained field access.

Reply:
There's a special test with `field_access_chain`, this is on ignoring primitives. Initially it wasn't a problem because field access was ignored, but now it would pollute the test.